option optimize_for = LITE_RUNTIME;

message LegionWorshipRequest
{
 required uint32 worship_id = 1;
}
